### ISSUE TITLE
Function name not updating

### ISSUE THREAD

**Issue by krishnaraovs** [movAX13h/P8Coder#4]
Whenever we are creating new functions the name of the function is showing the default text "newFunctiongame" instead of our choosen name.

Eg: if we created a function test then it should appear as
function test()
end

but instead of that its appearing as

function newFunctiongame()
end

**Comment by movAX13h** [comment]
Yes, thank you, I know. It's because the entry (and its code) is created right in the moment when you click the button and before you can enter a name. I will rework this when I got some time for it.
